server/src/main/java/com/scalar/db/server/service/ServerModule.java

```diff
@@ -77,23 +77,23 @@ private void startJmxReporter(MetricRegistry metricRegistry) {
     Runtime.getRuntime().addShutdownHook(new Thread(reporter::stop));
   }
 
-  private void startPrometheusHttpEndpoint(MetricRegistry metricRegistry) {
-    int prometheusHttpEndpointPort = config.getPrometheusHttpEndpointPort();
-    if (prometheusHttpEndpointPort == 0) {
+  private void startPrometheusExporter(MetricRegistry metricRegistry) {
+    int prometheusExporterPort = config.getPrometheusExporterPort();
+    if (prometheusExporterPort < 0) {
       return;
     }
 
     CollectorRegistry.defaultRegistry.register(new DropwizardExports(metricRegistry));
 
-    Server server = new Server(prometheusHttpEndpointPort);
+    Server server = new Server(prometheusExporterPort);
     ServletContextHandler context = new ServletContextHandler();
     context.setContextPath("/");
     server.setHandler(context);
     context.addServlet(new ServletHolder(new MetricsServlet()), "/metrics");
```

Comment:
I made Scalar DL's endpoint to match with the one of Envoy.
https://www.envoyproxy.io/docs/envoy/latest/operations/admin#get--stats-prometheus

```suggestion
    context.addServlet(new ServletHolder(new MetricsServlet()), "/stats/prometheus");
```